requirements.txt

```diff
@@ -47,6 +47,6 @@ xlrd==2.0.1
 pytest==7.4.0
 pytest-test-groups==1.0.3
 pytest-xdist==3.3.1
-flojoy==0.1.10
+flojoy==0.1.11
```

Comment:
Note for reviewers: this is required for proper integration within `studio` to use `flojoy.display` (tests are failing on `main` for this reason currently).